.circleci/config.yml

```diff
@@ -13,14 +13,14 @@ jobs:
           key: dependency-cache-{{ checksum "package.json" }}
```

Comment:
check yarn.lock rather then the package.json as its less likely to change and checks for changes in dep